docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -680,4 +680,81 @@ Like [hasSubsequence](#hasSubsequence) but assumes `haystack` and `needle` are U
 
 ## hasSubsequenceCaseInsensitiveUTF8
 
-Like [hasSubsequenceUTF8](#hasSubsequenceUTF8) but searches case-insensitively.
\ No newline at end of file
+Like [hasSubsequenceUTF8](#hasSubsequenceUTF8) but searches case-insensitively.
+
+## byteHammingDistance
+
+Calculates the hamming distance between two byte strings.
```

Comment:
The official notion in the docs is "ASCII string".

Reply:
@rschu1ze But this is incorrect.

ASCII is 7-bit encoding. Our strings are encoding-agnostic.

Reply:
Okay, I was wrong indeed. We usually only speak of "strings" and "UTF-8 strings". So, "byte strings" should be okay.